src/components/analyser/search/DashboardCards.tsx

```diff
@@ -66,52 +69,51 @@ export const DashboardCardsRow: React.FC<DashboardCardsRowProps> = ({
   stock,
   selectStock,
 }) => {
-  const [hover, setHover] = React.useState<boolean>(false);
   const classes = useStyles();
   const { t } = useTranslation();
 
   return (
-    <Card
-      className={classes.card}
-      style={{ width: 280, height: 450, margin: 10 }}
-    >
-      <CardMedia
-        style={{
-          height: 200,
-          width: 260,
-          placeItems: 'center',
-          alignItems: 'center',
-          objectFit: 'scale-down',
-          margin: 10,
+    <Card className={classes.card} style={{ width: 280, margin: 10 }}>
```

Comment:
move width and margin to styles section (const useStyles)

Reply:
done